src/python/WMCore/Services/DBS/DBS3Reader.py

```diff
@@ -971,3 +971,32 @@ def getParentDatasetTrio(self, childDataset):
                     frozenKey = frozenset(runLumiPair)
                     parentFrozenData[frozenKey] = fileId
         return parentFrozenData
+
+    def getDBSStatus(self, dataset):
+        """
+        The function to get the DBS status of outputs
+        :param dataset: dataset name
+        :return: DBS status of the given dataset
+        """
+
+        allowedDbsStatuses = ["VALID", "INVALID", "PRODUCTION"]
```

Comment:
a dataset can have other statuses. If I'm not wrong, DELETED and DEPRECATED as well. However, I don't think we need to have a list of valid statuses, we just use whatever is provided by DBS.

Reply:
As agreed, this extra check is also removed.